wordpress.org/public_html/wp-content/plugins/wporg-gp-custom-errors/wporg-gp-custom-errors.php

```diff
@@ -0,0 +1,208 @@
+		}
+
+		return esc_html__( 'Must be one of the following integers: 0, 1, or 6.', 'glotpress' );
+	}
+
+	/**
+	 * Adds an error for unexpected slug format.
+	 *
+	 * @param string      $original    The original string.
+	 * @param string      $translation The translated string.
+	 * @param GP_Original $gp_original The GP_original object.
+	 * @param GP_Locale   $locale      The locale.
+	 * @return string|true The error message or true if no error.
+	 */
+	public function error_unexpected_slug( $original, $translation, $gp_original, $locale ) {
+		if ( ! $this->is_core_project( $gp_original ) ) {
+			return true;
+		}
+		if ( is_null( $gp_original->context ) ) {
+			return true;
+		}
+		if (
+			strpos( $gp_original->context, 'Default post slug' ) === false &&
+			strpos( $gp_original->context, 'sample permalink structure' ) === false
+		) {
+			return true;
+		}
+		$spaces_present = $translation !== trim( $translation );
+		if ( $translation == sanitize_title( $translation ) ) {
+			return true;
+		}
+
+		if ( $spaces_present ) {
+			return esc_html__( 'The translation has empty spaces, new lines or another similar elements.', 'glotpress' );
+		}
+
+		return sprintf(
+		/* translators: %1$s: The slug made with the translation suggested. */
+			esc_html__( 'Must be a slug, like %1$s.', 'glotpress' ),
+			sanitize_title( $translation )
+		);
+	}
+
+	/**
+	 * Adds an error for unexpected date formats.
+	 *
+	 * @param string      $original    The original string.
+	 * @param string      $translation The translated string.
+	 * @param GP_Original $gp_original The GP_original object.
+	 * @param GP_Locale   $locale      The locale.
+	 * @return string|true The error message or true if no error.
+	 */
+	public function error_timezone_date_format( $original, $translation, $gp_original, $locale ) {
+		if ( ! $this->is_core_project( $gp_original ) ) {
+			return true;
+		}
+		if ( is_null( $gp_original->context ) ) {
+			return true;
+		}
+		if ( ! str_contains( $gp_original->context, 'timezone date format' ) ) {
+			return true;
+		}
+
+		$date_time = DateTime::createFromFormat( $translation, gmdate( $translation ) );
+		if ( $date_time ) {
+			return true;
+		}
+
+		$spaces_present = $translation !== trim( $translation );
+		if ( $spaces_present ) {
+			return esc_html__( 'The translation has empty spaces, new lines or another similar elements.', 'glotpress' );
+		}
+
+		return esc_html__( 'Must be a valid timezone date format.', 'glotpress' );
+	}
+}
+
+/**
+ * Returns the instance of the WPorg_GP_Custom_Translation_Errors class.
+ *
+ * @return WPorg_GP_Custom_Translation_Errors
+ */
+function wporg_gp_custom_translation_errors() {
+	 global $wporg_gp_custom_translation_errors;
+
+	if ( ! isset( $wporg_gp_custom_translation_errors ) ) {
+		$wporg_gp_custom_translation_errors = new WPorg_GP_Custom_Translation_Errors();
+	}
+
+	return $wporg_gp_custom_translation_errors;
+}
+
+add_action( 'plugins_loaded', 'wporg_gp_custom_translation_errors' );
```

Comment:
When changing this, you don't need the plugins_loaded priority hack
```suggestion
add_action( 'init', 'wporg_gp_custom_translation_errors' );
```

Reply:
It works on production with `plugins_loaded`